### ISSUE TITLE
Entering text does not work in android voice access

### ISSUE THREAD

**Issue by chunhtai** [flutter/flutter#77271]
## Steps to Reproduce
```dart
import 'package:flutter/material.dart';
void main() {
  runApp(
    MaterialApp(
      home: Scaffold(
        appBar: AppBar(
          title: Text('Expansion Tile Splash'),
          centerTitle: true,
        ),
        body: TextField(),
      ),
    ),
  );
}
```

1. In android download and turn on voice access
2. In the app, speak "start editing"
3. speak "type how are you"

expect: "how are you" is typed into the text field
actual: nothing happened
